test_passgithelper.py

```diff
@@ -55,6 +59,47 @@ def test_handle_skip_exits(monkeypatch: Any) -> None:
         passgithelper.handle_skip()
 
 
+def test_password_store_dir_selection(monkeypatch: Any) -> None:
```

Comment:
Can we split this up into multiple test cases for the different conditions tested here?

Reply:
Keeping all of those together helps understanding how the different variants interact. At least that's what I thought. But if you prefer to have them in separate tests, I can make that change.

Reply:
I think these make perfect individual test cases such as `uses the environment variable as the store dir if defined` or `uses the default directory when nothing is configured by the user` or `ignores empty store dir variables`.

So I'd be more in favor of that style.

Reply:
Done